flink-connectors/flink-connector-jdbc/src/test/java/org/apache/flink/connector/jdbc/catalog/PostgresCatalogTestBase.java

```diff
@@ -289,4 +292,29 @@ public static TestTable getArrayTable() {
 			));
 	}
 
+	public static TestTable getSerialTable() {
+		return new TestTable(
+			TableSchema.builder()
+				.field("f0", DataTypes.SMALLINT())
+				.field("f1", DataTypes.INT())
+				.field("f2", DataTypes.SMALLINT())
+				.field("f3", DataTypes.INT())
+				.field("f4", DataTypes.BIGINT())
+				.field("f5", DataTypes.BIGINT())
+				.build(),
+			"f0 smallserial, " +
+				"f1 serial, " +
+				"f2 serial2, " +
+				"f3 serial4, " +
+				"f4 serial8, " +
+				"f5 bigserial",
+			"0," +
+				"1," +
+				"2," +
+				"3," +
+				"4," +
+				"5"
```

Comment:
This values is never used.
How about add a ITCase that read serial type data properly to check the convert logic? and we can use more meaningful values like "32767, 2147483647, 32767, 2147483647 , 9223372036854775807, 9223372036854775807" .

Reply:
What do you mean exactly? Should I just use the suggested values or do sonething else?

Reply:
I mean we can add an ITCase that read serial type from PG, and the serial type value could be suggested values.

Reply:
Probably now I understood..you mean that for this table there's only the test that verifies the schema, not the returned values..I'll add it right now

Reply:
Just one note..now I rebase the PR and addressed your comments..however I also had to set notNull() on the fields because serial fields are marked as not null by PG. Let me know if you think I should handle this situation in a different way

Reply:
It's fine to me, we only read serial from PG and wont write a serial column.